csharp/test/Microsoft.ML.OnnxRuntime.Tests.MAUI/Microsoft.ML.OnnxRuntime.Tests.MAUI.csproj

```diff
@@ -136,7 +136,15 @@
 			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
 			<Visible>false</Visible>
 		</None>
-	</ItemGroup>
+    <None Include="$(OnnxRuntimeRoot)\..\onnxruntime\test\testdata\lora\two_params_lora_model.onnx">
```

Comment:
Why it starts with  `$(OnnxRuntimeRoot)\..` ? 
Should it be `$(OnnxRuntimeCSharpRoot)\..` ?
OnnxRuntimeRoot is `$(OnnxRuntimeCSharpRoot)\..`

Reply:
I think it should be `$(OnnxRuntimeRoot)\onnxruntime\...'.

Using `$(OnnxRuntimeCSharpRoot)\..` seems unnecessarily indirect when `$(OnnxRuntimeRoot)` is available.